### DISCUSSION TITLE
Lack of activity

### DISCUSSION THREAD

**Discussion by darrenburns** [darrenburns/posting#101] [Announcements]
Hi folks - just a quick update in case anyone is wondering why I haven't been very active with development and responding to issues/discussions recently.

I've got a strain injury in one of my fingers, which means I need to try and take it a bit easy on the coding front. 

Updates to Posting will continue at a considerably slower pace until I recover 😄 

Thanks!

**Comment by xavierog** [comment]
> I've got a strain injury in one of my fingers

That sounds painful -- get well soon!

> I need to try and take it a bit easy on the coding front.

Does that mean you are going to watch Highlander (1986)?

**Reply by darrenburns** [reply]
Thanks Xavier! 

> Does that mean you are going to watch Highlander (1986)?

Absolutely not.
